src/components/Logo/Logo.module.css

```diff
@@ -1,6 +1,6 @@
 .logo {
     color: rgb(145, 0, 0);
-    margin: 0px 10px;
+    margin: 0px 1rem;
     width: 120px;
     font-family: 'Brush Script MT';
     font-size: large;
```

Comment:
Why rem and not em?
In the video that I sent you there is a good explanation when to use each one.
In general, we should use rem on mainly (and maybe only) on font size, as it is saying something like ```"From this point forward, the scale of everything is changing"```, so go through your code and adjust properly, as you used rem where you should probably use em.